src/pages/BitcoinDashboardV2/components/PendingWithdraws/PendingWithdraws.tsx

```
@@ -0,0 +1,127 @@
+const tokens = {
+  bitcoin: {
+    Light: BitcoinIcon,
+    Dark: BitcoinIcon
+  } as Icons,
+  oraichain: {
+    Light: OraiLightIcon,
+    Dark: OraiDarkIcon
+  } as Icons
+};
+
+export const PendingWithdraws: React.FC<{ btcAddresses?: String[] }> = ({ btcAddresses }) => {
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)